src/main/scala/io/iohk/ethereum/faucet/jsonrpc/FaucetMethodsImplicits.scala

```diff
@@ -0,0 +1,25 @@
+package io.iohk.ethereum.faucet.jsonrpc
+
+import io.iohk.ethereum.faucet.jsonrpc.FaucetDomain.{SendFundsRequest, SendFundsResponse, StatusRequest, StatusResponse}
+import io.iohk.ethereum.jsonrpc.JsonMethodsImplicits
+import io.iohk.ethereum.jsonrpc.JsonRpcError.InvalidParams
+import io.iohk.ethereum.jsonrpc.serialization.JsonMethodDecoder.NoParamsMethodDecoder
+import io.iohk.ethereum.jsonrpc.serialization.{JsonEncoder, JsonMethodDecoder}
+import org.json4s.JsonAST.{JArray, JObject, JString}
+
+object FaucetMethodsImplicits extends JsonMethodsImplicits {
```

Comment:
How about a name that would tell us more what exactly those implicits do e.g. `FaucetJsonCodecs` ?

Reply:
Right, is true, I will change the name in the next task 252.